zkvm/docs/zkvm-spec.md

```diff
@@ -828,6 +787,39 @@ V == v·B + 0·B2
 1. Prover shows `v`.
 2. Verifier checks equality `V == v·B`.
 
+### Taproot
+Based on Gregory Maxwell's [Taproot proposal](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-January/015614.html), Taproot provides efficient and privacy-preserving storage of smart contracts. Multi-party blockchain contracts typically have a top-level "success clause" (all parties agree and sign) or "alternative clauses" to let a party exit the contract based on pre-determined constraints. Taproot has three significant features.
```

Comment:
Writing style nit: don't start the paragraph with a dependent clause, state the topic instead. ("Taproot provides efficient and privacy-preserving etc. It is based on [a proposal by Greg Maxwell](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-January/015614.html).")